great_expectations/data_context/store/store.py

```diff
@@ -55,6 +55,15 @@ def _validate_key(self, key):
     def store_backend(self):
         return self._store_backend
 
+    @property
+    def store_backend_id(self) -> str:
+        """
+        Report the store_id of the currently-configured StoreBackend
+        Returns:
+            store_id which is a UUID(version=4)
+        """
+        return self._store_backend.store_id
```

Comment:
Per our conversation, I'm wondering if we should call this a `store_backend_id` rather than a `store_id`, since it sounds like it's possible for multiple stores to share a single store_backend.
```suggestion
        return self._store_backend.store_backend_id
```

Reply:
This makes sense to me, would you mind weighing in here @jcampbell in case there is some reason I'm not thinking about?

Reply:
I agree that multiple stores will report the same id. 

With respect to the concrete name: both frankly work for me, since it's kinda doing everything (*by design*) it's the id of the store backend, which then becomes the id of the store. Defer to convo you two had.